smart-contracts/tasks/deploy/pool.ts

```diff
@@ -133,7 +146,10 @@ task('deploy:pool', 'Deploy a relay pool')
           }
           const originAssetContract = new ethers.Contract(
             originAsset,
-            ['function asset() view returns (address)'],
+            [
+              'function asset() view returns (address)',
+              'function symbol() view returns (string)',
```

Comment:
Not needed

Reply:
on L151 right below we call `symbol()` and without the abi that was failing for me

Reply:
But that's not the rigth symobol! The symbol of the asset is not the same as the symbol of the pool!

Reply:
We also a;ready have       const assetSymbol = await assetContract.symbol() and       const assetName = await assetContract.name() above.